features/step_definitions/volunteer_op_steps.rb

```diff
@@ -25,7 +25,9 @@
 end
 
 Then(/^there should be (\d+) doit volunteer ops stored$/) do |count|
-  expect(VolunteerOp.count(source: 'doit')).to eq count.to_i
+  sql = "SELECT COUNT(*) FROM volunteer_ops WHERE source = 'doit'"
+  doit_ops = VolunteerOp.count_by_sql(sql)
```

Comment:
yes, probably - it was `VolunteerOp.count(source: 'doit')` and that was throwing an error, where it worked before ... I just ended up trying to work out what the proper use of count with AR was now and that led me to count_by_sql which is damn ugly